docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/cards/media--cgov-infographic--multimedia-card.html.twig

```diff
@@ -4,9 +4,10 @@
 {% else %}
   {% set title = content.field_browser_title|field_value %}
 {% endif %}
+{% set infographic_url = media.path.0.alias %}
```

Comment:
This needs to call the URL Generator to get the path. This still exibits the issue where a Spanish infographic embedded on a spanish page will be missing the /espanol/ part of the path. That is because the alias is not enough - additionally fetching the alias directly does not allow any of the PathProcessors we have created to fire off.

Reply:
Actually, this is slightly incorrect, I see this is for home/ landing. Embedded Infographics have the same issue.

Reply:
This is loosely related to #2052 in that the same approach was taken for:
* https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/media/infographic/media--cgov-infographic--infographic-display-article-medium.html.twig
* https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/media/infographic/media--cgov-infographic--infographic-display-article-large.html.twig